src/js/loginHandler.js

```diff
@@ -0,0 +1,24 @@
+const logoutButton = document.querySelector('#logout');
+
+const getCookies = () => {
+    return document.cookie.split('; ').reduce((prev, current) => {
+        const [name, ...value] = current.split('=');
+        prev[name] = value.join('=');
+        return prev;
+    }, {});
+}
+
+function delete_cookie( name ) {
+    let cookies = getCookies();
+
+    if(Object.keys(cookies).includes(name)) {
+        document.cookie = name+"=";

```

Comment:
We can try to get a package to do that for us, but this project has different serving for multiple things so... yeah.